qa/suites/orch/cephadm/no-agent-workunits/task/test_orch_cli.yaml

```diff
@@ -6,6 +6,10 @@ roles:
   - mon.a
   - mgr.a
   - client.0
+overrides:
+  ceph:
+    log-only-match:
+      - CEPHADM_
```

Comment:
this test failed with `(CEPHADM_PAUSED)` which we actually need to ignore, as this is expected